### PR TITLE
fix edit name workspaces

### PR DESCRIPTION
**Pull request by sanglevinh**

### CODE REVIEW

File: frontend/src/pages/Workspace/index.tsx [hunk 1]
```diff
@@ -366,13 +367,14 @@ const Workspaces = () => {
     setInitName(params.row.name)
   }
 
-  const onCellClick: GridEventListener<'cellClick'> | undefined = (event) => {
+  const onCellClick: GridEventListener<'cellClick'> | undefined = (event: any) => {
```
Comment (itutu-tienday) [inline frontend/src/pages/Workspace/index.tsx:370]: It appears that edits are not confirmed even in the following cases.

- Example of operation)
  1. edit the workspace name of the record in the first row. 
  2. select a column other than workspace name in a record other than the first row.
      - The edit is supposed to be confirmed, but the edit is canceled.

If the above can be easily corrected, please do so. (Low priority).

File: frontend/src/pages/Workspace/index.tsx [hunk 2]
```diff
@@ -366,13 +367,14 @@ const Workspaces = () => {
     setInitName(params.row.name)
   }
```
Comment (itutu-tienday) [inline frontend/src/pages/Workspace/index.tsx:369]: One more correction is noted for #101. Please make the correction.

- #101 
  - > Adjust message in delete confirmation dialog

**Review by itutu-tienday — CHANGES_REQUESTED** [review]
@sanglevinh 
Some pointers are noted. Please check them.

File: frontend/src/pages/Workspace/index.tsx (hunk 1 above)
Comment (sanglevinh) [reply frontend/src/pages/Workspace/index.tsx:383]: @itutu-tienday I fixed it, can you check it for me?

File: frontend/src/pages/Workspace/index.tsx (hunk 2 above)
Comment (sanglevinh) [reply frontend/src/pages/Workspace/index.tsx:382]: @itutu-tienday i already added it

**Review by itutu-tienday — APPROVED** [review]

**Pull request merged**